jsk_arc2017_baxter/euslisp/lib/baxter-interface.l

```diff
@@ -225,9 +259,46 @@
             ((eq type :pinch)
              (send self :update-robot-state :wait-until-update t)
              (let ((finger-av (send self :get-real-finger-av l/r))
-                   (prev-av (send robot :angle-vector)) avs)
+                   prev-av av avs hand-interpolatingp)
+               ;; rotate arm using proximity sensor
                (setf (aref finger-av 1) 180)
-               (send self :move-hand l/r finger-av 1000)
+               (send self :move-hand l/r finger-av 2000 :wait nil)
+               (unix:usleep 10000)
+               (while (and
+                       (< (max (send self :get-proximity l/r :right)
+                               (send self :get-proximity l/r :left))
+                          proximity-threshold-)
+                       (setq hand-interpolatingp (send self :hand-interpolatingp))))
+               (when hand-interpolatingp
+                 (send self :cancel-move-hand)
+                 (send self :update-robot-state :wait-until-update t)
+                 (setq prev-av (send robot :angle-vector))
+                 (if (< (send self :get-proximity l/r :right) (send self :get-proximity l/r :left))
+                     (progn
+                        (send robot l/r :wrist-r :joint-angle 45 :relative t)
+                        (setq av (send robot :angle-vector))
+                        (send robot :angle-vector prev-av)
+                        (send self :angle-vector-raw av 2000)
+                        (send self :move-hand l/r finger-av 4000 :wait nil)
+                        (unix::usleep 10000)
+                        (while (and (< (send self :get-proximity l/r :right)
+                                       (send self :get-proximity l/r :left))
+                                    (send self :interpolatingp))))
```

Comment:
I found `:wait-interpolation-until` also forgot sleeping
https://github.com/start-jsk/jsk_apc/blob/master/jsk_arc2017_baxter/euslisp/lib/baxter-interface.l#L375-L378
I'll fix this in another PR.